tools/ci_build/build.py

```diff
@@ -2552,7 +2554,7 @@ def main():
                 )
 
     cmake_extra_defines = normalize_arg_list(args.cmake_extra_defines)
-    cross_compiling = args.arm or args.arm64 or args.arm64ec or args.android
+    cross_compiling = is_arm_cross_compile(args) or args.android
```

Comment:
This var impacts when tests can be run. 
So, 
If "--arm64" or "--arm64ec" was specified but the host CPU is not ARM64, this var should be true. Otherwise on Windows it should be always false.